e2e/cypress/integration/account_settings/sidebar/channel_switcher_spec.js

```diff
@@ -211,4 +211,48 @@ describe('Account Settings > Sidebar > Channel Switcher', () => {
         // * Channel name should be visible in LHS
         cy.get(`#sidebarItem_${testChannel.name}`).should('be.visible');
     });
+
+    it('Cmd/Ctrl+Shift+L closes Channel Switch modal and sets focus to post textbox', () => {
+        // # Go to a known team and channel
+        cy.visit('/ad-1/channels/town-square');
+        cy.get('#channelHeaderTitle').should('be.visible').should('contain', 'Town Square');
+
+        // # Type CTRL/CMD+K
+        cy.get('#post_textbox').cmdOrCtrlShortcut('K');
+
+        // * Channel switcher hint should be visible
+        cy.get('#quickSwitchHint').should('be.visible').should('contain', 'Type to find a channel. Use ↑↓ to browse, ↵ to select, ESC to dismiss.');
+
+        // # Type CTRL/CMD+shift+L
+        cy.get('#quickSwitchInput').cmdOrCtrlShortcut('{shift}L');
+
+        // * Suggestion list should be visible
+        cy.get('#suggestionList').should('not.be.visible');
+
+        // * focus should be on the input box
+        cy.get('#post_textbox').should('be.focused');
+        cy.get('#post_textbox').should('be.empty');
+    });
+
+    it('Cmd/Ctrl+Shift+M closes Channel Switch modal and sets focus to mentions', () => {
+        // # Go to a known team and channel
+        cy.visit('/ad-1/channels/town-square');
+        cy.get('#channelHeaderTitle').should('be.visible').should('contain', 'Town Square');
+
+        // # Type CTRL/CMD+K
+        cy.get('#post_textbox').cmdOrCtrlShortcut('K');
+
+        // * Channel switcher hint should be visible
+        cy.get('#quickSwitchHint').should('be.visible').should('contain', 'Type to find a channel. Use ↑↓ to browse, ↵ to select, ESC to dismiss.');
+
+        // # Type CTRL/CMD+shift+m
+        cy.get('#quickSwitchInput').cmdOrCtrlShortcut('{shift}M');
+
+        // * Suggestion list should be visible
+        cy.get('#suggestionList').should('not.be.visible');
+
+        // * searchbox should appear
+        cy.get('#searchBox').should('have.attr', 'value', '@ @user-1 ');
```

Comment:
This will fail in case other test changes the the "words that trigger mentions".  I suggests to make predefined by patching the test user.  It can be in the `before` hook or inside this test block (before `cy.visit`).
```javascript
cy.apiPatchMe({notify_props: {first_name: 'false', mention_keys: 'user-1'}});
```

Reply:
added to the specific test, as it shouldn't be needed for any other